srcs/app/config/CommonConfig.cpp

```diff
@@ -2,5 +2,6 @@
 
 CommonConfig::CommonConfig()
 	: root("html"), client_max_body_size(1000000), autoindex(false),
-	index("index.html") {
+	index("index.html"), error_pages(), upload_store(""), return_status(),
```

Comment:
This is just a question. It is needed to add to the initialisation list attributes that won't be initialised?

Reply:
Great question! I think it's not required because they will be default or zero-initialized.

https://en.cppreference.com/w/cpp/language/default_initialization#Notes